test_runner.py

```diff
@@ -118,15 +137,5 @@ def run_test(test_flavor: OverrideDefinitions, full_path: str):
             config = tomllib.load(f)
             is_integration_test = config["job"].get("use_for_integration_test", False)
             if is_integration_test:
-                test_flavors = [OverrideDefinitions()] + integration_tests_flavors[
-                    config_file
-                ]
-
-                for test_flavor in test_flavors:
+                for test_flavor in integration_tests_flavors[config_file]:
                     run_test(test_flavor, full_path)
-
-                    # Deleting checkpoint folder from test
```

Comment:
Not sure why we did this in the first place.  The ckpts were already in separate dirs so they wouldn't interfere.  And now that I am preserving output artifacts, it makes sense to keep them.